server/service/src/sync/sync_on_central/mod.rs

```diff
@@ -113,9 +67,62 @@ pub async fn pull(
     .map(SyncRecordV6::from)
     .collect();
 
+    warn!("Sending records as central server: {:#?}", records);
+
     Ok(SyncBatchV6 {
         total_records,
         end_cursor,
         records,
     })
 }
+
+/// Receive Records from a remote open-mSupply Server
+pub async fn push(
+    service_provider: &ServiceProvider,
+    SyncPushRequestV6 {
+        batch,
+        sync_v5_settings,
+    }: SyncPushRequestV6,
+) -> Result<SyncPushSuccessV6, SyncParsedErrorV6> {
+    use SyncParsedErrorV6 as Error;
+
+    warn!("Push!: {:#?}", batch);
+    // TODO consolidate at top level ? As middleware ?
```

Comment:
We have the middleware now, but is it still good to have this as a reminder you're in the `central server` code?